modules/js/perf/loader.js

```diff
@@ -0,0 +1,32 @@
+function errorLog(log) {
+    console.log(log);
+}
+
+var openCVLoader = async function (binPath, onloadCallback) {
+    OPENCV_URL = "opecv.js"
+    let simdSupported = await wasmFeatureDetect.simd();
```

Comment:
it may need to test whether wasm is supported at all. For that case, it should load the asm.js build.